db/migrate/20221006124530_retire_anndata_and_seurat_feature_flag.rb

```diff
@@ -0,0 +1,16 @@
+  class RetireAnndataAndSeuratFeatureFlag < Mongoid::Migration
+    def self.up
+      FeatureFlag.retire_feature_flag('upload_seurat_and_anndata')
+      FeatureFlag.retire_feature_flag('upload_seurat_and_anndata')
```

Comment:
Blocker: this is likely a typo, but you've already retired the feature flag, so calling this twice throws an error (see below, Jean also encountered this error).  Also, in the `down` section, you're creating two instances of the `FeatureFlag` which would also throw an error.
```
==  RetireAnndataAndSeuratFeatureFlag: migrating ==============================
rails aborted!
StandardError: An error has occurred, 20221006124530 and all later migrations canceled:

FeatureFlag: 'upload_seurat_and_anndata' does not exist

    raise NameError, "FeatureFlag: '#{name}' does not exist" if feature_flag.blank?
    ^^^^^
/Users/bistline/Documents/Rails/single_cell_portal_core/app/models/feature_flag.rb:32:in `retire_feature_flag'
/Users/bistline/Documents/Rails/single_cell_portal_core/db/migrate/20221006124530_retire_anndata_and_seurat_feature_flag.rb:4:in `up'

Caused by:
NameError: FeatureFlag: 'upload_seurat_and_anndata' does not exist

    raise NameError, "FeatureFlag: '#{name}' does not exist" if feature_flag.blank?
    ^^^^^
/Users/bistline/Documents/Rails/single_cell_portal_core/app/models/feature_flag.rb:32:in `retire_feature_flag'
/Users/bistline/Documents/Rails/single_cell_portal_core/db/migrate/20221006124530_retire_anndata_and_seurat_feature_flag.rb:4:in `up'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)

```